clients/src/main/java/org/apache/kafka/clients/admin/internals/CoordinatorStrategy.java

```diff
@@ -98,7 +98,7 @@ public ApiRequestScope lookupScope(CoordinatorKey key) {
                 key.idValue + "' cannot be represented in a request."));
         }
         FindCoordinatorResponse response = (FindCoordinatorResponse) abstractResponse;
-        if (batch) {
+        if (!response.data().coordinators().isEmpty()) {
```

Comment:
could we add a comment here to explain what case it is (ex: batch response case)